xlr/converters/src/test-helpers.ts

```diff
@@ -1,5 +1,5 @@
-import ts from 'typescript';
-import tsvfs from '@typescript/vfs';
+import * as ts from 'typescript';
+import * as tsvfs from '@typescript/vfs';
```

Comment:
Noticed that test-helpers was only being used in the xlr-converters.  So I just moved it into that package instead.  It's not imported in the index.ts so it won't be bundled.   Probably don't need `* as`  in the import anymore, but I added it just incase. When vfs was part of the build it would complain it can't find `default` in the package.